README.md

```diff
@@ -316,6 +316,30 @@ When set to false, when clicking on a rail, the click event will be allowed to p
 When set to true, you can scroll the container by selecting text and move the cursor.  
 **Default: false**
 
+### theme
```

Comment:
Oh, yes, right. I thought it was a section title :open_mouth: 
Please keep it as it is.